Sources/Helpers/OAuthHelper.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  OAuthHelper.swift
+//  OsmAnd Maps
+//
+//  Created by nnngrach on 15.05.2023.
+//  Copyright © 2023 OsmAnd. All rights reserved.
+//
+
+// OAuth version: 2.0
+// Grant type: Authorization code
+
+import SwiftUI
+import Foundation
+import AuthenticationServices
+
+@objc class OAuthHelper : NSObject {
+
+    //Override in sublcasses
+    internal class var authURL: String { return "" }
```

Comment:
internal is the default protection level, there's no need to specify it explicitly